trio/_core/_exceptions.py

```diff
@@ -103,3 +104,16 @@ class ResourceBusyError(Exception):
     the data get scrambled.
 
     """
+
+
+class ClosedResourceError(Exception):
+    """Raised when attempting to use a resource after it has been closed.
+
+    Note that "closed" here means that *your* code closed the resource,
+    generally by calling a method with a name like ``close`` or ``aclose``. If
```

Comment:
Do you think it's worth also mentioning the "exited a context manager" possibility for why the thing was closed?

Reply:
Yeah, that's a good idea!